src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,10 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private static final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        return userService.findByEmail(email) != null && userService.findByEmail(email)
```

Comment:
Can you call the method findByEmail only once? Don’t complicate the performance of your solution ;)